overrides/scripts/ThermalExpansion.zs

```diff
@@ -389,7 +389,7 @@ recipes.addShaped(<thermalexpansion:frame>, [
 [<gregtech:meta_item_1:12183>,<gregtech:meta_item_1:12183>,<gregtech:meta_item_1:12183>],
 [<gregtech:meta_item_1:12183>,<thermalexpansion:frame:64>,<gregtech:meta_item_1:12183>],
 [<gregtech:meta_item_1:12183>,<gregtech:meta_item_1:12183>,<gregtech:meta_item_1:12183>]]);
-<thermalexpansion:frame:64>.displayName = "Thermal Machine Casing";
+<thermalexpansion:frame>.displayName = "Thermal Machine Casing";

```

Comment:
This incorrectly labels "Machine Frame" as "Thermal Machine Casing".

`<thermalexpansion:frame:64>` and `<thermalexpansion:frame>` are two different items.

Reply:
> This incorrectly labels "Machine Frame" as "Thermal Machine Casing".
> 
> `<thermalexpansion:frame:64>` and `<thermalexpansion:frame>` are two different items.

`<thermalexpansion:frame:64>` was originally Device Frame, which was overwritten by **L399** `<thermalexpansion:frame:64>.displayName = "Thermal Device Casing";` with Thermal Device Casing.

Judging from that, <thermalexpansion:frame>, with its original displayed name Machine Frame, is supposed to be tweaked to Thermal Machine Casing.

Now, see L392 and L399. L392 overwrites the name of <thermalexpansion:frame:64> with Thermal Machine Casing, and L399 double overwrites the name of exactly the same item with Thermal Device Casing, that makes zero sense. If this commit incorrectly labels "Machine Frame" as "Thermal Machine Casing", then I don't get why L399, to which I didn't make any change, correctly labels "Device Frame" as "Thermal Device Casing".

Reply:
@NotMyWing :nudge: